DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Helpers/RevitConversionContextStack.cs

```diff
@@ -0,0 +1,22 @@
+using Autodesk.Revit.DB;
+using Speckle.Converters.Common;
+
+namespace Speckle.Converters.RevitShared.Helpers;
+
+[System.Diagnostics.CodeAnalysis.SuppressMessage(
+  "Naming",
+  "CA1711:Identifiers should not have incorrect suffix",
+  Justification = "See base class justification"
+)]
+public class RevitConversionContextStack : ConversionContextStack<UI.UIDocument, ForgeTypeId>
+{
+  public const double TOLERANCE = 0.0164042; // 5mm in ft
+
+  public RevitConversionContextStack(RevitContext context, IHostToSpeckleUnitConverter<ForgeTypeId> unitConverter)
+    : base(
```

Comment:
Is it ok to throw in this constructor? Since this entire thing will be dependency injected, when would we be expecting the Document to be null, only on Revit startup?